tf2onnx/graph.py

```diff
@@ -462,7 +462,7 @@ def __init__(self, nodes, output_shapes=None, dtypes=None, target=None, opset=No
         self._output_to_consumers = {}
         self._input_to_graph = {}
         self.shapes = {}
-        self.graph_name = graph_name or "tf2onnx"
+        self.graph_name = graph_name or utils.make_name("tf2onnx")
```

Comment:
Did you determine the cause of the identical names issue?

Reply:
yes, its the main graph that is set to 'tf2onnx' when writing the onnx file, which is actually what we want.